config/main.py

```diff
@@ -789,6 +791,86 @@ def del_vlan_member(ctx, vid, interface_name):
     db.set_entry('VLAN_MEMBER', (vlan_name, interface_name), None)
 
 
+
```

Comment:
Please remove extra line

Reply:
Done. Removed extra lines between the new definitions that got added for mvrf.